backend/config.js

```diff
@@ -13,6 +13,7 @@ const base = {
         region: 'us-east-1'
     },
     awsBucket : "your-aws-bucket",
+    indabaDomain: process.env.INDABA_CLIENT_URL || 'http://localhost:3000',
```

Comment:
I think we can make the meaning of this property more clear by calling it `clientDomain` or `indabaClientDomain`